src/azure-cli/azure/cli/command_modules/iot/_validators.py

```diff
@@ -53,7 +53,13 @@ def validate_feedback_ttl(ns):
 def validate_feedback_lock_duration(ns):
     if (ns.feedback_lock_duration and
             ns.feedback_lock_duration not in range(5, 301, 1)):
-        raise ArgumentError(None, 'Please specify the duration from 5 to 300 seconds only.')
+        raise ArgumentError(None, 'Please specify the feedback lock duration from 5 to 300 seconds only.')
```

Comment:
Please use the specific error type `InvalidArgumentValueError` instead of `ArgumentError`